objects/src/accounts/delta.rs

```diff
@@ -1,5 +1,11 @@
 use super::{AccountStorage, Felt};
-use crate::crypto::merkle::{MerkleStoreDelta, MerkleTreeDelta};
+use crate::{
+    assets::Asset,
+    crypto::{
+        merkle::{MerkleStoreDelta, MerkleTreeDelta},
+        utils::collections::Vec,
+    },
+};
 use assembly::ast::ModuleAst;
```

Comment:
nit: this can be replaced with:
```Rust
use super::{AccountStorage, Felt};
use crate::{
    assembly::ModuleAst,
    assets::Asset,
    crypto::merkle::{MerkleStoreDelta, MerkleTreeDelta},
    utils::collections::Vec,
};
```